src/Microsoft.ML.AutoML/Experiment/Runners/CrossValRunner.cs

```diff
@@ -30,6 +31,7 @@ internal class CrossValRunner<TMetrics> : IRunner<CrossValidationRunDetail<TMetr
             IEstimator<ITransformer> preFeaturizer,
             ITransformer[] preprocessorTransforms,
             string labelColumn,
+            string groupIdColumn,
```

Comment:
nit: Alphabetical order within same type?

Reply:
Is that a dotnet coding guideline? If so, can you please share a pointer?


---
In reply to: [442461900](https://github.com/dotnet/machinelearning/pull/5246#discussion_r442461900) [](ancestors = 442461900)